addons/account/models/account_move.py

```diff
@@ -3029,6 +3029,19 @@ def check_field_access_rights(self, operation, field_names):
             result = [fname for fname in result if fname not in weirdos]
         return result
```

Comment:
```suggestion
        field_names = field_names or self._get_default_read_fields()
        return super().check_field_access_rights(operation, field_names)
```

Reply:
We are aligning the check access methods and we need a fst one for checking whether a user can read/write a specific field.
You have a few more lines of code, because you have 2 models in accounting that behave differently from rest of Odoo: you don't show some accessible fields when you `read()` without specifying the field names. Note that `check_field_access_rights` will disappear after the next major release when we clean deprecated methods.